backend/frontend/templates/frontend/auth/page_login.html

```diff
@@ -1,12 +1,35 @@
 {% extends "frontend/auth/base.html" %}
 {% block auth_heading %}Login{% endblock %}
 {% block auth_content %}
-  <form action="" method="post">
+  <form action="" method="post" class="my-4" novalidate>
     {% csrf_token %}
-    {{ form.as_p }}
-    <div class="my-3">
+
+    {% if form.non_field_errors %}
+      <ul class="my-4 {{styles.alert_error}}">
+        {{ form.non_field_errors.as_ul }}
+      </ul>
+    {% endif %}
+
+    <div class="my-4" >
       <a class="{{ styles.link }}" href="{% url 'user_forgot_password' %}">Forgot Password?</a>
     </div>
+
+    {% for field in form %}
+      <label for="{{ field.id_for_label }}" class="{{ styles.label_small }}">{{ field.label_tag }}</label>
+      {{ field }}
+      {% if field.errors %}
+          <ul class="{{ styles.field_error }}">
+              {% for error in field.errors %}
+                  <li>
+                      <i class="fa-solid fa-triangle-exclamation"></i>
```

Comment:
This is not DRY. And I'm sure that if we had to write this kind of thing out manually for every single form we build forever then that would turn into a big mess. We need to centralise functionality and use the tools at our disposal.

There are 2 options here:
1. Look here: https://docs.djangoproject.com/en/5.0/topics/forms/#reusable-form-templates OR
2. Upgrade the ThemedFormMixin so it does all the rendering

Currently it seems that setting up a form template is the way to go